vue-components/stories/Button.stories.ts

```diff
@@ -60,30 +60,30 @@ export function iconOnly(): Component {
 				<div>
 					<div style="margin-bottom:16px;">
 						<Button variant="normal" type="neutral" iconOnly aria-label="More Information">
-							<Icon type="info" size="large" color="inherit" />
+							<Icon type="info" size="large" color="inherit" style="vertical-align: top;" />
```

Comment:
So, in a way, we're removing the alignment properties from the icon component, but then we're applying them to it again inline, in the context of the parent. The responsibility is not fully removed. I wonder, could the alignment be defined within the button component code? Meaning, making sure that whatever is inserted in the button slot(s) has a default center alignment (there are no exceptions to this).

Reply:
In case this comment is blocking (I'm aware there's a QB PR depending on this one) we could take this as an early step and explore how to ensure proper alignment of icons within components in a separate task.

Reply:
> Meaning, the button is wired in a way that makes whatever is inserted in its slot(s) has a default center alignment

But, that is not the case. Text is correctly and by default aligned at baseline, as it should. But images need to be aligned top (or bottom, that barely makes a difference).

Reply:
yep, I know it's not the case. I was trying to ask if that alignment should be defined within the component's code (like in the case of the validation message?) instead of in the story.

Reply:
Yes, it might make sense to discuss that in an extra task. It would be speculative CSS in the button, probably trying to match `.wikit-Icon`

Reply:
@itamargiv just a quick check: do you agree to merging this PR as is and creating an extra task? 

Wondering if this could be related to/covered by [T270327](https://phabricator.wikimedia.org/T270327)